package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "@raha.io/website",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "scripts": {
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
+    "start": "yarn pre-build && react-scripts-ts start",
+    "build": "yarn pre-build && react-scripts-ts build",
```

Comment:
Can we just run all of pre, build, and post here? No one seems to be calling `yarn build` on its own.